diff --git a/dlls/user32/user32.spec b/dlls/user32/user32.spec
index 95cef3914f3..3b2d007498c 100644
--- a/dlls/user32/user32.spec
+++ b/dlls/user32/user32.spec
@@ -1108,7 +1108,7 @@
 # @ stub SetWindowCompositionTransition
 @ stdcall SetWindowContextHelpId(long long) NtUserSetWindowContextHelpId
 @ stdcall SetWindowDisplayAffinity(long long)
-# @ stub SetWindowFeedbackSetting
+@ stdcall SetWindowFeedbackSetting(long long long long ptr)
 @ stdcall SetWindowLongA(long long long)
 @ stdcall -arch=win64 SetWindowLongPtrA(long long long)
 @ stdcall -arch=win64 SetWindowLongPtrW(long long long)
diff --git a/dlls/user32/win.c b/dlls/user32/win.c
index cef0beeacda..86c8f640f88 100644
--- a/dlls/user32/win.c
+++ b/dlls/user32/win.c
@@ -1837,3 +1837,13 @@ WORD WINAPI TileChildWindows( HWND parent, UINT flags )
 {
     return TileWindows( parent, flags, NULL, 0, NULL );
 }
+
+/**********************************************************************
+ *              SetWindowFeedbackSetting (USER32.@)
+ */
+BOOL WINAPI SetWindowFeedbackSetting( HWND hwnd, FEEDBACK_TYPE feedback, 
+        DWORD dwFlags, UINT32 size, const void *configuration)
+{
+     FIXME("(%p, %p): im a stub :3 \n", hwnd, configuration);
+    return TRUE;  
+}
